src/egon/data/datasets/fix_ehv_subnetworks.py

```diff
@@ -113,15 +118,26 @@ def add_line(x0, y0, x1, y1, v_nom, scn_name, cables):
     if v_nom == 220:
         s_nom = 520
         x_per_km = 0.001 * 2 * np.pi * 50
+        r_per_km = 0.097  # based on average r from similar lines
+        b_per_km = 3.066e-6  # based on average b from similar lines
+        cost_km = 40.697  # based on average costs from similar lines
```

Comment:
I am not quite sure where it is normally set. But also here should be somewhere a value or calculation which could be directly used.

Reply:
Change applied.